doc/source/usage.rst

```diff
@@ -16,14 +16,14 @@ Run example case as a Nextflow pipeline (recommended)
 
 The *ksrates* pipeline can be automatically run through Nextflow with a few preparation steps.
 
-1.  Access in a terminal the directory that will host the rate-adjustment results (assumed here to be ``example``) and unzip the sequence data files in there:: ::
+1.  Access in a terminal the directory that will host the rate-adjustment results (assumed here to be ``example``) and unzip the sequence data files in there::
 
         cd ksrates/example
         gunzip elaeis.fasta.gz oryza.fasta.gz asparagus.fasta.gz elaeis.gff3.gz
 
 2.  Prepare the configuration files.
 
-    The directory already contains a pre-filled *ksrates configuration file* (``config_elaeis.txt``) and a *Nextflow configuration file* template (``custom_nextflow.config``). If running on a cluster or within a container, please fill in the template as described in the :ref:`nextflow_config_section` section.
+    The directory already contains a pre-filled *ksrates configuration file* (``config_elaeis.txt``) and a *Nextflow configuration file* template (``custom_nextflow.config``) to be filled in as described in the :ref:`nextflow_config_section` section.
```

Comment:
Would it make sense to rename the custom_nextflow.config to simply the default nextflow.config? Otherwise we should add the "-c custom_nextflow.config" to all nextflow command lines everywhere in the documentation (e.g. also in Installation -> Testing your installation).

Reply:
Now that it is a "mandatory" file it makes much sense to do that. By renaming it just nextflow.config it will be automatically detected and used, and we can drop the "-c" flag in the launching command.
Shall I delete it from everywhere then? I'll just note down in the Usage that you can explicitly give the path with "-c" if you want to call your file another way or if you want to give a specific path.

Reply:
Sounds good to me, just make sure that it's still described that it is needed and should be called nextflow.config.